### PR TITLE
feat: Increase highlight characters limit

### PR DESCRIPTION
**Pull request by ir3ne**
## Description

This PR increase the highlight characters limit.
In the second commit a space after a class is removed because not necessary.

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Closes: #1365 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by jpmcb — APPROVED** [review]
LGTM - looks like some workflow checks failed. And curious what others think about a length of 300.

**Review by babblebey — APPROVED** [review]
LGTM 🍕

**Comment by babblebey** [comment]
> LGTM - looks like some workflow checks failed. And curious what others think about a length of 300.

I think its great, I had once been limited by the character length, but sometimes I thought it was just a fine idea to have had it limited, it kept the sanity of just keeping it short with the title.

Plus, I think the length will have some effect of the OG generated Image too for Highlights.

**Comment by bdougie** [comment]
> Plus, I think the length will have some effect of the OG generated Image too for Highlights.

This.

I would love to see examples of titles that need 300 chars. The goal for the highlight is to provide a space to showcase your that is not a blog post or twitter.

Not everyone developer is an influencer and a short concise is what is encouraged.

**Comment by OgDev-01** [comment]
>Not everyone developer is an influencer and a short concise is what is encouraged

Considering the social cards, anything greater than the length of 50 for now will lead to UI breaks or having to truncate the title in the social card which IMO is not a good approach. I think we stick to max 50 `char` for now pending when an updated UI that can accommodate more than 50 `char` (if there is one in the future)

File: components/molecules/HighlightInput/highlight-input-form.tsx [hunk 1]
```diff
@@ -106,12 +106,12 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
...
     <form onSubmit={handlePostHighlight} className="flex flex-col flex-1 gap-4">
       <Collapsible onOpenChange={setIsDivFocused} open={isDivFocused}>
-        <div className="flex flex-col gap-2 p-2 overflow-hidden text-sm bg-white border rounded-lg ">
+        <div className="flex flex-col gap-2 p-2 overflow-hidden text-sm bg-white border rounded-lg">
           <CollapsibleTrigger asChild>
             <div className="flex pr-2">
               <input
                 value={title}
-                maxLength={50}
+                maxLength={300}
```
Comment (bdougie) [inline components/molecules/HighlightInput/highlight-input-form.tsx:114]: Please returns this to 50. We need to considere more aspects of the UX before making a changing like this.

**Review by bdougie — CHANGES_REQUESTED** [review]

File: components/molecules/HighlightInput/highlight-input-form.tsx (hunk 1 above)
Comment (ir3ne) [reply components/molecules/HighlightInput/highlight-input-form.tsx]: @bdougie done.

**Comment by brandonroberts** [comment]
Can we close this? This would require API updates also, or else it would fail on submission of more than 50 characters.

**Comment by bdougie** [comment]
The increased limit on the title was the only change and per my feedback not idea. Closing for now.

**Pull request closed without merging**